components/BottomNavigation/examples/BottomNavigationTypicalUseExample.m

```diff
@@ -76,7 +76,12 @@ - (void)commonBottomNavigationTypicalUseExampleInit {
                                       tag:0];
   tabBarItem5.badgeValue = @"999+";
 #if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0)
-  tabBarItem5.badgeColor = [MDCPalette cyanPalette].accent700;
+#pragma clang diagnostic push
```

Comment:
I'm of the opinion that this isn't exactly what we want to be doing. Instead someone should go thru and add @available all over the place.

Reply:
@available is only available on iOS 11+, so we would have to do a macro check for iOS 11 inside the macro check for iOS 10 to see if we can use @available. If we can't use @available, we'd still need to check if the selector is defined since it was introduced in iOS 10.